### PR TITLE
Config options

### PR DESCRIPTION
**Pull request by chereseeriepa**
- set port otherwise defaults to `26835`
- enable cors (disabled by default)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -16,8 +17,10 @@ const createUnboxTransform = (queryParam) => {
   return createUnboxStream(keyBytes, zeros);
 }
 
-function ServeBlobs(sbot) {
+function ServeBlobs(sbot, config) {
   return function(req, res, next) {
+    const corsEnabled = get(config, 'serveBlobs.cors', false)
```
Comment (mixmix) [inline index.js]: put this below line 20 so its not run every request

Comment (chereseeriepa) [reply index.js]: oops

**Comment by staltz** [comment]
Hey @chereseeriepa and @mixmix, although I agree with this PR I think it was merged prematurely (happened within 4 hours total), I would have appreciated my request for a review before merging. I authored this module and I am still interested in maintaining it. In particular, I wouldn't have included `lodash.get` just for this change.

**Comment by mixmix** [comment]
Sorry about that @staltz 
I'd talked with @chereseeriepa about this out of band of this thread, had reviewed this, and encouraged us to move to publish this.
Happy to revert changes you're not stoked with.

**Pull request merged**